### DISCUSSION TITLE
Technical question about Sync

### DISCUSSION THREAD

**Discussion by M1k30n3** [Chia-Network/chia-blockchain#3681] [Support]
I an using the Windows version and in a brief time I see that it is Synced in green in the full node window and the Wallet is Syncing (always, never says synced).

I can see my farm in the console, the time to win of 2 years (funny)  and none of my plots validated.

Port scan shows 8444 closed , tried different things available online but port still closed. I am a tmobile subscriber with a mobile wifi hotspot because I have no other options.

However I see that most reports of not synced issues never state that the node sync and the wallet is syncing so I must ask. Do I really have the syncing problem or it could be something else. I only have 10 plots; I started yesterday; is not having plot validation normal?

Thanks for your replys

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
